akka-http/src/main/scala/akka/http/scaladsl/server/Rejection.scala

```diff
@@ -98,9 +98,9 @@ final case class MalformedHeaderRejection(headerName: String, errorMsg: String,
  * Rejection created by [[akka.http.scaladsl.server.directives.HeaderDirectives.checkSameOrigin]].
  * Signals that the request was rejected because `Origin` header value is invalid.
  */
-final case class InvalidOriginRejection(invalidOrigins: immutable.Seq[SHttpOrigin])
+final case class InvalidOriginRejection(allowedOrigins: immutable.Seq[SHttpOrigin])
```

Comment:
No need to put the illegal one into the response, however it should be in the rejection, as perhaps someone may want to write a handler that logs "from which illegal origins people try to access my server".